src/conf_mode/high-availability.py

```diff
@@ -31,6 +33,39 @@
 from vyos import airbag
 airbag.enable()
 
+
+def _is_ipv6_tentative(iface: str, ipv6_address: str) -> bool:
```

Comment:
Maybe a candidate for `vyos.utils.network`?

Reply:
I'm not sure what else may use it, but technically, yes. I also think that "run `/sbin/ip` with `--json`" should be a primitive in `vyos.utils.network`.

Reply:
Will do.

Reply:
@c-po  Done!